pkg/wekafs/apiclient/filesystem.go

```diff
@@ -135,6 +141,20 @@ func (a *ApiClient) DeleteFileSystem(r *FileSystemDeleteRequest) error {
 	return nil
 }
 
+func (a *ApiClient) GetFileSystemMountToken(r *FileSystemMountTokenRequest, token *FileSystemMountToken) error {
+	f := a.Log(5, "Getting a mount token for filesystem", r.Uid)
```

Comment:
why not constants for log levels?

Reply:
also, what's a poing of deferring this logline? 
also not sure..how comes `Log` function returns a function? Kinda does not make sense